toolkits/google/arcade_google/tools/drive.py

```diff
@@ -82,3 +88,96 @@ async def list_documents(
             break
 
     return {"documents_count": len(files), "documents": files}
+
+
+@tool(
+    requires_auth=Google(
+        scopes=["https://www.googleapis.com/auth/drive.file"],
+    )
+)
+async def get_file_tree_structure(
+    context: ToolContext,
+    include_shared_drives: Annotated[
+        bool, "Whether to include shared drives in the file tree structure. Defaults to True."
+    ] = True,
```

Comment:
I'm wary of defaulting to including shared drives as someone who is a part of a large organization with many shared drives could have tens of thousands of documents in those shared drives.

Up to you though - I see both pros and cons

Reply:
let's be safe and make it default to `False` then